SU2_CFD/src/output/output_structure_legacy.cpp

```diff
@@ -1249,34 +1249,33 @@ void COutputLegacy::SetConvHistory_Body(ofstream *ConvHist_file,
 
     bool Unsteady = ((config[val_iZone]->GetTime_Marching() == DT_STEPPING_1ST) ||
                      (config[val_iZone]->GetTime_Marching() == DT_STEPPING_2ND));
-    bool In_NoDualTime = (!DualTime_Iteration && (iExtIter % config[val_iZone]->GetWrt_Con_Freq() == 0));
-    bool In_DualTime_0 = (DualTime_Iteration && (iIntIter % config[val_iZone]->GetWrt_Con_Freq_DualTime() == 0));
+    bool In_NoDualTime = !DualTime_Iteration;
+    bool In_DualTime_0 = DualTime_Iteration;
     bool In_DualTime_1 = (!DualTime_Iteration && Unsteady);
-    bool In_DualTime_2 = (Unsteady && DualTime_Iteration && (iExtIter % config[val_iZone]->GetWrt_Con_Freq() == 0));
-    bool In_DualTime_3 = (Unsteady && !DualTime_Iteration && (iExtIter % config[val_iZone]->GetWrt_Con_Freq() == 0));
+    bool In_DualTime_2 = (Unsteady && DualTime_Iteration);
+    bool In_DualTime_3 = (Unsteady && !DualTime_Iteration);
 
     /*--- Header frequency: analogy for dynamic structural analysis ---*/
     /*--- DualTime_Iteration is a bool we receive, which is true if it comes from FEM_StructuralIteration and false from SU2_CFD ---*/
     /*--- We maintain the name, as it is an input of the function ---*/
-    /*--- The function GetWrt_Con_Freq_DualTime should be modified to be able to define different frequencies ---*/
     /*--- dynamic determines if the problem is, or not, time dependent ---*/
     bool dynamic = (config[val_iZone]->GetTime_Domain());              // Dynamic simulations.
-    bool In_NoDynamic = (!DualTime_Iteration && (iExtIter % config[val_iZone]->GetWrt_Con_Freq() == 0));
-    bool In_Dynamic_0 = (DualTime_Iteration && (iIntIter % config[val_iZone]->GetWrt_Con_Freq_DualTime() == 0));
+    bool In_NoDynamic = !DualTime_Iteration;
+    bool In_Dynamic_0 = DualTime_Iteration;
     bool In_Dynamic_1 = (!DualTime_Iteration && nonlinear_analysis);
-    bool In_Dynamic_2 = (nonlinear_analysis && DualTime_Iteration && (iExtIter % config[val_iZone]->GetWrt_Con_Freq() == 0));
-    bool In_Dynamic_3 = (nonlinear_analysis && !DualTime_Iteration && (iExtIter % config[val_iZone]->GetWrt_Con_Freq() == 0));
+    bool In_Dynamic_2 = (nonlinear_analysis && DualTime_Iteration);
+    bool In_Dynamic_3 = (nonlinear_analysis && !DualTime_Iteration);
 
     bool write_heads;
     if (Unsteady) write_heads = (iIntIter == 0);
-    else write_heads = (((iExtIter % (config[val_iZone]->GetWrt_Con_Freq()*40)) == 0));
+    else write_heads = true;
 
-    bool write_turbo = (((iExtIter % (config[val_iZone]->GetWrt_Con_Freq()*40)) == 0) || (iExtIter == (config[val_iZone]->GetnInner_Iter() -1)));
+    bool write_turbo = true;
 
     /*--- Analogous for dynamic problems (as of now I separate the problems, it may be worthy to do all together later on ---*/
     bool write_heads_FEM;
     if (nonlinear_analysis) write_heads_FEM = (iIntIter == 0);
-    else write_heads_FEM = (((iExtIter % (config[val_iZone]->GetWrt_Con_Freq()*40)) == 0));
+    else write_heads_FEM = true;
```

Comment:
Is there nothing equivalent for Wrt_Con_Freq?

Reply:
Sure ... SCREEN_WRT_FREQ_INNER

Sometimes I am not really clever, I mean... its late ... but still

Reply:
It is hard to resist not deleting that entire file xD